pkg/kor/all.go

```diff
@@ -378,6 +380,13 @@ func GetUnusedAllNonNamespaced(filterOpts *filters.Options, clientset kubernetes
 }
 
 func GetUnusedAll(filterOpts *filters.Options, clientset kubernetes.Interface, apiExtClient apiextensionsclientset.Interface, dynamicClient dynamic.Interface, outputFormat string, opts common.Opts) (string, error) {
+	if NamespacedFlagUsed {
+		if opts.Namespaced {
+			return GetUnusedAllNamespaced(filterOpts, clientset, outputFormat, opts)
+		}
+		return GetUnusedAllNonNamespaced(filterOpts, clientset, apiExtClient, dynamicClient, outputFormat, opts)
```

Comment:
Shouldn't this return also namespaced resources?

Reply:
No, the flow is as so:
1. Check is the flag is used
1.1. If it's used, check if it's true
1.1.1 If it's true, return namespaced resources
1.2. If it's not true, return non-namespaced resources
2. Return both (unless `--include-namespaces` is used)

Reply:
I think you set the default to true so it will return only namespaced resources if the flag is not used

Reply:
I'll try to make it clearer
The code assumes there are 3 different states
- flag is true
- flag is false
- flag does not exist

I think that because there is a default option 3 can never happen. I might be wrong but please check

Reply:
The code is aware to a scenario where the flag is not used, by passing the state through `cmd.Flags().Changed("namespaced")`, an example is shown in the description.